instrumentation/github.com/gin-gonic/gin/otelgin/gintrace.go

```diff
@@ -10,13 +10,14 @@ import (
 
 	"github.com/gin-gonic/gin"
 
-	"go.opentelemetry.io/contrib/instrumentation/github.com/gin-gonic/gin/otelgin/internal/semconvutil"
 	"go.opentelemetry.io/otel"
 	"go.opentelemetry.io/otel/attribute"
 	"go.opentelemetry.io/otel/codes"
 	"go.opentelemetry.io/otel/propagation"
 	semconv "go.opentelemetry.io/otel/semconv/v1.20.0"
 	oteltrace "go.opentelemetry.io/otel/trace"
+
+	"go.opentelemetry.io/contrib/instrumentation/github.com/gin-gonic/gin/otelgin/internal/semconvutil"
```

Comment:
Can you revert this import rearrange change? We are grouping `go.openteletry.io`. See https://github.com/open-telemetry/opentelemetry-go-contrib/blob/96b1ed1e7b0c1613290df1801ff6ba8474a33ce5/.golangci.yml#L108-L109